lms/static/scripts/frontend_apps/services/content-info-fetcher.js

```diff
@@ -43,6 +51,18 @@ export class ContentInfoFetcher {
       path: urlPath`/api/jstor/articles/${contentId.itemId}`,
     });
 
+    const links = /** @type ContentInfoLinks */ ({
```

Comment:
The code in this module maps stuff from the proxied API fields (`api-types` typing) to what we're going to send to the client (`client-rpc` typing). Included in that is mapping `related_items` entries over to URLs in a `links` object...

Reply:
If this is what we want we could include the links from the backend? There is an advantage to this as we'd use `route_url` which would automatically update if we ever changed the route. We won't, so it's probably a moot point, but you know... style points?

We could also do this for the JSTOR URL above.